controllers/submariner/route_agent_resources.go

```diff
@@ -89,7 +89,7 @@ func newRouteAgentDaemonSet(cr *v1alpha1.Submariner, name string) *appsv1.Daemon
 						}}},
 						// Path used by Openshift
 						{Name: "host-var-run-ovn-ic-nbdb-sock", VolumeSource: corev1.VolumeSource{HostPath: &corev1.HostPathVolumeSource{
-							Path: "/var/run/ovn-ic/ovnnb_db.sock",
+							Path: "/var/run/ovn-ic/ovnnb_db.sock", Type: ptr.To(corev1.HostPathFileOrCreate),
```

Comment:
I think we can apply this change to the other sockets too? WDYS @skitt @aswinsuryan

Reply:
yes I think , we would need that for removing existing bridges in the case of an upgrade from old implementation to new implementation.